app/views/comments/_commentbox.html.erb

```diff
@@ -31,8 +31,8 @@ data-shortid="<%= comment.short_id if comment.persisted? %>">
         </div>
       <% end %>
 
-      <%= f.submit "#{comment.new_record?? "Post" : "Update"}",
-        :class => "comment-post", :disabled => !@user %>
+      <%= f.button "#{comment.new_record?? "Post" : "Update"}",
```

Comment:
Why change this? I thought the rails method here is going to make an `<input type="submit">`, right?